salvo/src/lib/api/control.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+
+package salvo;
+
+import "src/lib/api/image.proto";
+import "src/lib/api/source.proto";
+import "src/lib/api/env.proto";
+
+
+message JobControl {
```

Comment:
Can we add a comment that describes the purpose of the `JobControl` message?

Reply:
Absolutely.  I'll add a comment